shared/GlobalContext.ts

```diff
@@ -26,5 +26,4 @@ export interface AppServices {
   request: ActorRefFrom<typeof requestMachine>;
   scan: ActorRefFrom<typeof scanMachine>;
   revoke: ActorRefFrom<typeof revokeVidsMachine>;
-  qrLogin: ActorRefFrom<typeof qrLoginMachine>;
```

Comment:
We have created the qrlogin machine following the revoke machine, here the revoke machine is the direct child to the app machine but in our case, QR login is not the child machine to the app machine. It is the child to scan machine once the qrlogin machine reaches its final state it has to communicate with the scan machine but because of the above app services it is communicating with the app machine which is causing issues in the scan machine it is not getting onDone event